python/build-wheel-manylinux2014.sh

```diff
@@ -96,6 +96,7 @@ for ((i=0; i<${#PYTHONS[@]}; ++i)); do
   # The -d flag removes directories. The -x flag ignores the .gitignore file,
   # and the -e flag ensures that we don't remove the .whl directory, the
   # dashboard directory and jars directory.
+  git config --global --add safe.directory /ray
```

Comment:
out of curiosity, why do we need to add /ray as a safe directory to git?  Is the /ray repo owned by another user?

Reply:
> out of curiosity, why do we need to add /ray as a safe directory to git? Is the /ray repo owned by another user?

Great question. As I mentioned earlier, this is git checkout action bug here https://github.com/actions/checkout/issues/760, and what happens is as this guy shared,



""

This happens if you have a different user who owned the directory. For example, your git repo is located in /var/www which is owned by www-data. Now, when you are signed-in/using a non-sudo user account and you go to /var/www to perform git actions such as "$ git branch" you will get this error so make sure you have appropriate directory permission

""

Currently, the way that I added there is a workaround (not the fix), and I'll need a separate fix on it.